tests/gcp/hooks/test_natural_language.py

```diff
@@ -22,7 +22,7 @@
 
 from google.cloud.language_v1.proto.language_service_pb2 import Document
 
-from airflow.contrib.hooks.gcp_natural_language_hook import CloudNaturalLanguageHook
+from airflow.gcp.hooks.natural_language import CloudNaturalLanguageHook
 from tests.compat import mock
 from tests.contrib.utils.base_gcp_mock import mock_base_gcp_hook_no_default_project_id
```

Comment:
You need to update import path in 46 lines and many other. Please look at Travis log.